pgdrive/obs/top_down_renderer.py

```diff
@@ -125,8 +132,97 @@ def render(self, vehicles, pheromone_map):
         else:
             self._draw_pheromone_map(pheromone_map)
             self._draw_vehicles(vehicles)
+        if data is not None:
+            # self._plot_data_curve(data)
+            self._plot_data_curve_raw(data)
         self.blit()
 
+    def _plot_data_curve_raw(self, data):
```

Comment:
Why the data plot function is in pheromone renderer? I think it can be called in all envs

Reply:
I thought it's strictly for pheromone rendering, can put it in base env tho.